src/components/workspace-utils.js

```diff
@@ -150,6 +155,43 @@ export const SnapshotInfo = ({
     length: { maximum: 63, tooLong: 'Name can\'t be more than 63 characters.' }
   })
 
+  const tdrErrorDisplay = () => div({ style: { paddingLeft: '1rem' } }, [
+    div({ style: Style.dashboard.header }, ['Error']),
+    `Details of this snapshot could not be loaded due to a problem calling Terra Data Repo: ${JSON.stringify(snapshotLoadError)}`
+  ])
```

Comment:
in the case of deleted snapshots, `snapshotLoadError` is an empty object and so this error message isn't super helpful. Again, since this is a corner case I'm not bothering to make it a lot better, and leaving this in place for other cases such as transient network errors.